serialize.go

```diff
@@ -514,6 +514,25 @@ func DeserializeQueryAndArray(context *Context, buffer *Buffer, serializationTyp
 	return array, query, nil
 }
 
+// HandleLoadArraySchemaRequest Passes the array and serialized LoadArraySchemaRequest to core which returns the
+// serialized LoadArraySchemaResponse. The request contains a TileDB Config used to load the schema, the response
+// contains the latest array schema loaded and a map of all array schemas.
+func HandleLoadArraySchemaRequest(array *Array, request *Buffer, serializationType SerializationType) (*Buffer, error) {
+	response, err := NewBuffer(array.context)
+	if err != nil {
+		return nil, fmt.Errorf("error creating LoadArraySchemaResponse buffer: %s", array.context.LastError())
+	}
+
+	ret := C.tiledb_handle_load_array_schema_request(array.context.tiledbContext, array.tiledbArray, C.tiledb_serialization_type_t(serializationType), request.tiledbBuffer, response.tiledbBuffer)
+	if ret != C.TILEDB_OK {
+		return nil, fmt.Errorf("error handling LoadArraySchemaRequset: %s", array.context.LastError())
+	}
+
+	runtime.KeepAlive(request)
+	runtime.KeepAlive(response)
```

Comment:
This really isn't necessary, since the buffer is returned, but isn't harmful either.

Reply:
Removed :+1: